### DISCUSSION TITLE
Why do we need to ingest a file to get answers

### DISCUSSION THREAD

**Discussion by ar-ohi-srivastav** [imartinez/privateGPT#623] [General]
From my limited knowledge what I can understand is that this repo asks us to download a pre-trained model and then helps us run queries to which responses are generated using the downloaded model. What I don't understand is that why are ingesting documents into this then

**Comment by kamaldua65** [comment]
Documents are sliced to small pieces of chunks. Then these chunks are converted to vector data. That is why you need to ingest

**Comment by milincheta** [comment]
The "theory" is that the pre trained model answers general questions, and has the basic language, lets say. Then you ingest your data, that is not know to the model mentioned and include that to be able to query that data.
I said theory because is not working for me :(
